manim/plugins/__init__.py

```diff
@@ -0,0 +1 @@
+# WARNING: THIS FILE IS AUTOGENERATED VIA `manim plugins -u`
```

Comment:
```suggestion
# WARNING: THIS FILE IS AUTOGENERATED VIA `manim plugins -u`
# Do not modify this unless you are absolutely certain you know what you are doing.
```

Reply:
This is definitely an area of concern, something that I would like to maybe address with the `plugins` subcommand to allows `from {plugin-name} import *` type imports instead of just `import {plugin-name}`